commands/apps/index.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+let co  = require('co');
+let cli = require('heroku-cli-util');
+let _   = require('lodash');
+
+function* run (context, heroku) {
+  let org = (!context.flags.personal && !context.flags.all && context.org) ? context.org : null;
```

Comment:
@dickeyxxx If you want this to match current behavior, you should not include `!context.flags.all` in this logic.  In the current behavior if someone passes both `--org` && `--all` the orgs endpoint is hit, not the personal one.